pyproject.toml

```diff
@@ -1,4 +1,5 @@
 [project]
+version = "0.1.0"
```

Comment:
Huh I don't know what you're thinking about the speed of changes.

I think the benefit of the version living in `__init__.py` is that it can be programmatically retrieved. If it isn't dynamic and just hardcoded in the pyproject.toml file, the version is only packaging metadata and not also a matter of inspectable code. (which is a bit of a lie, since there are ways to ask python to go look up the version of a module if it's been installed correctly - but that's still an `if`)